fe/fe-core/src/main/antlr4/org/apache/doris/nereids/DorisParser.g4

```diff
@@ -178,6 +221,12 @@ booleanValue
     : TRUE | FALSE
     ;
 
+aggFunction
+    : AVG '(' DISTINCT? expression ')'
```

Comment:
Is this a temporary solution? it is better not add any function name here.
If this is a temporary solution, please add a todo comment.

Reply:
done